Makefile

```diff
@@ -16,15 +16,15 @@ help:  ## Use one of the following instructions:
 
 .PHONY : lint
 lint : ## Static code analysis with Pylint
-	pylint -ry climada_petals > pylint.log || true
+	python -m pylint -ry climada_petals > pylint.log || true
 
 .PHONY : unit_test
 unit_test : ## Unit tests execution with coverage and xml reports
-	pytest $(PYTEST_ARGS) --ignore=climada_petals/test climada_petals/
+	python -m pytest $(PYTEST_ARGS) --ignore=climada_petals/test climada_petals/
```

Comment:
I see you updated the Jenkins scripts. Why was this addition necessary? `python -m pytest` as opposed to `pytest` adds the current directory to the `sys.path`. This enables testing local source code of modules which are not installed. At the same time, it makes it less clear what code exactly is tested. I recommend making sure that Petals is *installed* (possibly from sources with `pip install -e`), and then removing `python -m`, to make sure that the installed version is tested. See the [GitHub Actions](https://github.com/CLIMADA-project/climada_petals/blob/develop/.github/workflows/testing.yml) for reference, where `python -m` was not required so far

Reply:
The addition is necessary on Jenkins because all the branches use the same conda environment. `pip install -e` would interfere with other branches' tests.
For referring to an arbitrary climada core branch one can install that by creating a _venv_ environment on top of the _conda_ environment. (Which is about 10 times faster). But in such a layered environment `pytest` is taken from the underlying _conda_ sources and imports climada from there, while `python -m pytest` starts from _venv_ and imports climada from here.
This was done, e.g., for exactly this PR before #787 was merged.

GitHub Actions on the other hand build a new conda environment for every branch and commit. I'm afraid we don't have enough resources in terms of cpu or diskspace to do that on Jenkins.

Reply:
I am still confused why this is necessary here in Petals but not in Core. Doesn't that mean there is something off with the environment configuration?

Anyway, I see that this is too much hassle to change. However, I would like to keep the default invoking of `pytest` for users, which should work best with the current installation instructions. I found a way to insert the different command into the Makefile, and will push it shortly.

Reply:
Ok sorry I did not realize you added new bash scripts for Jenkins, see my comments there